raiden/tests/integration/network/transport/rif_comms/process.py

```diff
@@ -32,7 +32,7 @@ def start(env_name: str) -> Popen:
             preexec_fn=os.setsid,  # set to later kill process group
         )
 
-        # FIXME: we need some sort of ping call, this will sometimes not be enough
```

Comment:
I can suggest to use some sort of port process detection while you don't have the ping functionallity, that can be done by using a command call to lsof or something programatically.

Reply:
> I can suggest to use some sort of port process detection while you don't have the ping functionallity, that can be done by using a command call to lsof or something programatically.

i don't think this is good enough. the comms program will boot on a specific port, but it takes a few seconds before it registers the given RSK address, and in the meantime the port query will return `ok` (as i understand it).

it has to be programmed in the pub sub node in one way or another (synchronous start up, or separate ping call)